flusher.go

```diff
@@ -94,6 +94,12 @@ func (s *Server) FlushLocal(interval time.Duration, metricLimit int) {
 	// since not everything in tempMetrics is safe for sharing
 	go s.flushForward(tempMetrics)
 
+	go func() {
+		for _, p := range s.getPlugins() {
+			go p.Flush(finalMetrics, s.Hostname)
```

Comment:
should probably be reporting the same metrics as global instances? we can punt on refactoring that code out if you prefer